src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private ColorSupplier colorSupplier;
+    private Random random = new Random();
+    private final int magician = 10;
+    private final int lownum = 1;
```

Comment:
These vars are not constants as they have to be
And think about names. magician? 10 is magician? What is it? 10 is the **default radius**.
lownum - What is it? MIN_SIDE_LENGTH for example.
![image](https://github.com/mate-academy/jv-oop-advanced/assets/20326858/e8c9329a-1840-454d-bc72-78673d2dfbab)